bigbluebutton-html5/imports/ui/components/toast/component.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+
+import Icon from '../icon/component';
+import styles from './styles';
+
+const propTypes = {
+  icon: PropTypes.string,
+  message: PropTypes.node.isRequired,
+  type: PropTypes.oneOf(Object.values(toast.TYPE)).isRequired,
+};
+
+const defaultProps = {
+  icon: null,
+};
+
+const defaultIcons = {
+  [toast.TYPE.INFO]: 'help',
+  [toast.TYPE.SUCCESS]: 'checkmark',
+  [toast.TYPE.WARNING]: 'warning',
+  [toast.TYPE.ERROR]: 'close',
+  [toast.TYPE.DEFAULT]: 'about',
```

Comment:
Are there cases where we need a default one?

Reply:
Every time the user didn't pass a custom icon.

Reply:
What custom icon are you talking about?
This icon can be selected only if you explicitly pass `type={'default'}`, as far as I can see.
I'm just curious if we need it, or you just included whatever was provided by `react-toastify`.

Reply:
Ah, I got it, never mind.
ToastContainer always passes `type={'default'}` unless we overwrite it with another value.